CVCotM: Clarify the Wii U VC version is unsupported (#4734)
ArchipelagoMW/Archipelago: worlds/cvcotm/docs/en_Castlevania - Circle of the Moon.md

which is currently a requirement to connect to a multiworld. This happens because all audio code was stripped
from the ROM, and all sound is instead played by the collection through external means.

> The Wii U Virtual Console version is also tested, but due to much of the code being shifted it doesn't patch well at all without
> breaking things pretty badly. As such, it is completely unsupported at the moment.

Reviewer: ```suggestion
The Wii U Virtual Console version does not work due to changes in the code in that version.
```
Just making it much more succinct

Author: Probably better that way, yeah.
